docs/docs/ecosystem/README.md

```diff
@@ -1,38 +0,0 @@
----
-sidebar_position: 1
----
-
-# Ecosystem 
```

Comment:
we should make an issue to build a larger ecosystem page on the docs website.

Reply:
Agree, but isn't it what awesome-cosmos or https://cosmos.network/ecosystem ought to be?

What the ecosystem page from the Cosmos SDK should look like? The way we had it was duplicating content.